wagtailmedia/static/wagtailmedia/js/media-chooser-telepath.js

```diff
@@ -0,0 +1,18 @@
+class MediaChooser {
+    constructor(html, idPattern) {
```

Comment:
Oh ok - I just copied the code from [here](https://github.com/wagtail/wagtail/blob/main/client/src/entrypoints/images/image-chooser-telepath.js) :). But it will probably be transpiled to ES5 so it's still compatible with IE.

Reply:
yeah, that bit gets transpiled. plus it is easier for core drop support in a new version whereas contrib modules are not always use with the latest and greatest =)